### ISSUE TITLE
Crashes trying to load any .vif

### ISSUE THREAD

**Issue by travisfe** [bigianb/bgda-explorer#20]
I'm using game files from the steam version. I can see skeletons from .anm files fine, but the exe crashes if I ever click on a .vif file. Also .tex files dont show anything. Seems like this was made before the steam version, are the files different?

Here's where it's breaking:

![image](https://user-images.githubusercontent.com/22872726/180672363-2251d10f-f9e6-4af9-921c-95fc56c41c79.png)

**Comment by RoseIlanga** [comment]
Yeah, this currently only works on the PS2 files, which are different from the PC (Steam/GOG on Windows, Mac OS, and Linux) files. The Xbox files seem to be identical to the PC ones, interestingly enough.
